include/ttmlir/Dialect/TTIR/Pipelines/TTIRPipelines.h

```diff
@@ -31,12 +32,28 @@ struct StableHLOToTTIRPipelineOptions
       // that the TTIR inliner pass may inline the ops.
       llvm::cl::init(true)};
 };
+#endif
+
+struct LinalgToLLVMPipelineOptions
+    : public PassPipelineOptions<LinalgToLLVMPipelineOptions> {
+  // TODO (vwells): we might want some more options to say lower through affine
+  // loops instead of scf directly, etc. which could be new options
+  Option<bool> cleanupOutputEnabled{
+      *this, "enable-post-pipeline-cleanup",
```

Comment:
I somewhat preferred the remove dead values name since that's effectively what most of these passes are doing.  That said, perhaps the name `enable-optimization-passes` might be more suitable?  This current name is a bit vague.

Reply:
Yeah, I personally agree that `remove-dead-values` seemed like a reasonable summary, but don't feel super strongly.  I think `enable-optimization-passes` is fine too, I'll switch to that